lib/ansible/modules/cloud/azure/azure_rm_aks.py

```diff
@@ -77,6 +77,10 @@
             os_disk_size_gb:
                 description:
                     - Size of the OS disk.
+            vnet_subnet_id:
+                description:
+                    - The Subnet ID for a existing Subnet or if you use a different CIDR for the Nodes IP (Kubenet or Azure CNI)
```

Comment:
I find this confusing. I've made a suggestion as a way to start the conversation - let me know if it's accurate.

```suggestion
                    - The subnet for the agent pool profile. For existing subnet, use subnet ID. For new subnet, use CIDR notation.
```